src/components/Header/header.js

```diff
@@ -0,0 +1,25 @@
+import { Link, Routes, Route } from 'react-router-dom';
+import './header.css';
+import MyCalculator from '../calculator/calculator';
+import MyQuotes from '../quotes/quotes';
+import Home from '../home/home';
+
+const Header = () => (
+  <>
+    <nav>
+      <h1>Math magicians</h1>
+      <ul className="link-holder">
+        <li className="links"><Link to="/">Home</Link></li>
+        <li className="links"><Link to="/calculator">Calculator</Link></li>
+        <li className="links"><Link to="/quote">Quote</Link></li>
+      </ul>
+    </nav>
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/calculator" element={<MyCalculator />} />
+      <Route path="/quote" element={<MyQuotes />} />
+    </Routes>
```

Comment:
- [ OPTIONAL ] I suggest that to keep your code well organized, it is better to move the routes to the App.js file. It is better to let the header only have navigation links. This will ensure that the Header component only has code that is relevant to the navigation and not the project routing. What do you think 🤔 ?